permits/templates/permits/_geo_time_form.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n %}
+{% load bootstrap4 %}
+
+<div class="card mb-3">
+  <div class="card-body">
+    <a data-toggle="collapse" href="#planning-{{ form_id }}">
```

Comment:
For this collapse who be easier to understand if we add arrow down & up. So it's easier to understand for the end-user the collapse action. 

On the same topic it's not really nice if a collapse is close when the user try to go to the next step but there is some error inside the form. The errors on the fields are hidden inside the collapse content.

Reply:
I agree it would be nice to have arrows here, but we don’t have icons on the projet yet and it’s not in the acceptance criteria, so I’d say it’s good enough for now, and we’ll add some once we add icons to the project.

The collapse is now open if the form has errors.